src/client/datascience/jupyter/kernels/kernel.ts

```diff
@@ -533,20 +527,6 @@ export class Kernel implements IKernel {
             );
         }
     }
-
-    private async updateRemoteUriList(serverConnection: INotebookProviderConnection) {
```

Comment:
This should go into the notebook provider, so that we have this when ever we connect to a remote jupyter server